CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Basic project setup
 cmake_minimum_required(VERSION 3.5)
 project(cpp-statsd-client
-        VERSION 1.0.2
+        VERSION 2.0.0
```

Comment:
Can we wait for a release to be prepared?

When we're ready to release v2, I'll open a PR and perform this change.

Reply:
yep of course, no problem, i wasnt sure how you like to handle that! thanks